conf/vsc_ugent.config

```diff
@@ -12,13 +6,10 @@ try {
     System.err.println("WARNING: Could not run sinfo to determine current cluster, defaulting to doduo")
 }
 
-if(!params.tier1_project && hostname.contains("dodrio")){
-    System.err.println("Please specify your project with --tier1_project in your Nextflow command or with params.tier1_project in your config file.")
-    System.exit(1)
-}
+def tier1_project = System.getenv("SBATCH_ACCOUNT") ?: System.getenv("SLURM_ACCOUNT")
```

Comment:
Both of these don't seem to contain the project name for our starting grant. Maybe we should return to the previous way with the params but you can set those environment variables as defaults instead

Reply:
These environment variables SLURM users have to define themselves (like I added in the docs and e.g [here](https://docs.dkrz.de/blog/2017/how-to-set-the-default-slurm-project-account.html)). This is the [official way](https://slurm.schedmd.com/sbatch.html#SECTION_INPUT-ENVIRONMENT-VARIABLES) to pass the --account option to SLURM, there is no need for a --tier1_project option.